src/flagsmith/mod.rs

```diff
@@ -385,24 +405,87 @@ mod tests {
     use httpmock::prelude::*;
 
     static ENVIRONMENT_JSON: &str = r#"{
-            "api_key": "B62qaMZNwfiqT76p38ggrQ",
-            "project": {
-                "name": "Test project",
-                "organisation": {
-                    "feature_analytics": false,
-                    "name": "Test Org",
-                    "id": 1,
-                    "persist_trait_data": true,
-                    "stop_serving_flags": false
+        "api_key": "B62qaMZNwfiqT76p38ggrQ",
+        "project": {
```

Comment:
Feels kinda icky to have `ENVIRONMENT_JSON` in this rust file. Can we not load it from a json file?

Reply:
It's surprisingly verbose, see https://stackoverflow.com/questions/66340266/generating-constants-ar-compile-time-from-file-content